src/Components/SelectLanguagePage/SelectLanguagePage.tsx

```diff
@@ -0,0 +1,78 @@
+import { FormControl, Select, InputLabel, MenuItem, SelectChangeEvent, Button, Box } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+import { Context } from '../Wrapper/Wrapper.tsx';
+import { useContext, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import languageOptions from '../../Assets/languageOptions.tsx';
+
+const SelectLanguagePage = () => {
+  const context = useContext(Context);
+  const { formData } = context;
```

Comment:
```suggestion
  const { formData } = useContext(Context);
```
I typically do this instead of the 2 line method.